src/components/Tool/FixPhoneNumbers/Contact.tsx

```diff
@@ -346,212 +357,26 @@ const Contact: React.FC<Props> = ({
-                                  <Box
-                                    display="flex"
-                                    justifyContent="center"
-                                    alignItems="center"
-                                  >
-                                    <Box
-                                      display="flex"
-                                      alignItems="center"
-                                      data-testid={`delete-${person.id}-${phoneNumber.id}`}
-                                      onClick={() =>
-                                        handleDeleteNumberOpen({
-                                          personId,
-                                          phoneNumber,
-                                        })
-                                      }
-                                      className={classes.ContactIconContainer}
-                                    >
-                                      <Tooltip
-                                        title={t('Delete Number')}
-                                        placement="left"
-                                      >
-                                        <Icon
-                                          path={mdiDelete}
-                                          size={1}
-                                          className={classes.hoverHighlight}
-                                        />
-                                      </Tooltip>
-                                    </Box>
-                                  </Box>
-                                ) : (
-                                  <Box
-                                    display="flex"
-                                    justifyContent="center"
-                                    alignItems="center"
-                                  >
-                                    <Box
-                                      display="flex"
-                                      alignItems="center"
-                                      data-testid={`lock-${person.id}-${index}`}
-                                      className={classes.paddingX}
-                                    >
-                                      <Icon
-                                        path={mdiLock}
-                                        size={1}
-                                        style={{
-                                          color:
-                                            theme.palette.cruGrayMedium.main,
-                                        }}
-                                      />
-                                    </Box>
-                                  </Box>
-                                )}
-                              </Box>
-                            </Grid>
-                          </>
-                        )}
-                      </Formik>
-                    ))}
+                    <FieldArray
```

Comment:
Can you help me understand why this is a `<FieldArray>` since we're not using `arrayHelpers`? I'm wondering if it could just be a regular `values.numbers.map(...)`.